pytorch_lightning/core/optimizer.py

```diff
@@ -60,17 +60,40 @@ def __init__(self,
         self._trainer = None
         self._optimizer = optimizer
         self._accumulate_grad_batches = accumulate_grad_batches
```

Comment:
```suggestion
        self.accumulate_grad_batches = accumulate_grad_batches
```

Reply:
Actually I think the initial value should be set to the private attribute or at least the attribute should be initialised first